src/main/java/seedu/bankwithus/Account.java

```diff
@@ -57,11 +59,16 @@ public void subtractBalance(float currentBalance, float withdrawal) {
         withdrawalChecker.updateTotalAmtWithdrawn(withdrawal);
     }
 
-    public String getName() {
-        return name;
+    public void setSaveGoal(SaveGoal saveGoal, String args, String untilWhenStr) {
+
+        this.saveGoal = saveGoal;
+        Ui.showSaveGoalCreated(args, untilWhenStr);
```

Comment:
Erm, this is almost definitely supposed to be ui.showSaveGoalCreated, rather than Ui.showSaveGoalCreated. In addition, I recommend that you do the printing in the parser class and let this function be just a general setSaveGoal function so that it is easier to use